airbyte-ci/connectors/metadata_service/lib/metadata_service/models/generated/ConnectorMetrics.py

```diff
@@ -0,0 +1,24 @@
+# generated by datamodel-codegen:
+#   filename:  ConnectorMetrics.yaml
+
+from __future__ import annotations
+
+from typing import Any, Optional
+
+from pydantic import BaseModel, Extra
+from typing_extensions import Literal
+
+
+class ConnectorMetrics(BaseModel):
+    all: Optional[Any] = None
```

Comment:
just a suggestion: `overall`? trying to think about how one would talk about these different metrics. "overall success rate, overall usage" - just a preference but not a super strong one

Reply:
Here i'm a bit confused about if this is generally reserved for all versions over all platforms, as it looks in the upstack pr, or just on all platforms (i guess on latest versions), as I assumed when reading this pr first

Reply:
It basically means "aggregate of all other values"

Ill ping Alex Gron with this :)

Reply:
Haha sorry for all the design feedback from someone who was not involved in that part of the process 😅 take it or leave it